connectivity/mbedtls/source/x509_crt.c

```diff
@@ -80,7 +80,11 @@
 #if !defined(_WIN32) || defined(EFIX64) || defined(EFI32)
 #include <sys/types.h>
 #include <sys/stat.h>
+#if defined(__MBED__)
+#include <platform/mbed_retarget.h>
+#else
 #include <dirent.h>
+#endif /* __MBED__ */
```

Comment:
Should be in a specific PR ?

Reply:
Probably, I inserted it here because 6a1c6b9058c3b51d6a3913f13dfb1c1133e26914 depends on it but we can spun it out

Reply:
Standalone PR created https://github.com/ARMmbed/mbed-os/pull/13863